pubtools/exodus/task.py

```diff
@@ -88,6 +91,14 @@ def main(self):
         with task_context():
             # setup the logging as required
             self._setup_logging()
-
-            self.run()
+            if self.exodus_enabled():
+                self.run()
+            else:
+                LOG.debug(
+                    "Please see the pubtools-exodus configuration documentation: %s",
+                    DOCUMENTATION_LINK,
+                )
+                raise RuntimeError(
+                    "Environment variable '%s' is not set" % "EXODUS_ENABLED"
```

Comment:
Isn't it also possible that EXODUS_ENABLED is falsy?
Probably more accurate just to say Exodus is disabled.